package-lock.json

```diff
@@ -2,7 +2,6 @@
   "name": "ICDC-Frontend",
```

Comment:
I don't see bento-components updated in here?

Reply:
Updated.